### ISSUE TITLE
Neuron patches for scenery

### ISSUE THREAD

**Issue by phet-steele** [phetsims/neuron#120]
@jbphet neuron needs to be patched from two scenery issues:

Canvas renderer not properly set on Edge for navbar/menu/homescreen phetsims/scenery/issues/515
(This is the main fix for phetsims/joist/issues/321)

Text boundsMethod 'fast' should still allow Canvas rendering (but prevent fitting) phetsims/scenery/issues/516 which fixes the concentration layout:
![neuron](https://cloud.githubusercontent.com/assets/5863899/13156029/bd80338e-d63d-11e5-9078-ab5696c95023.PNG)

**Comment by jbphet** [comment]
@jonathanolson and I discussed this and decided that instead of trying to patch Scenery so that bounds method values of 'fast' still allowed canvas, I would just try turning off that bounds method and using the default and qualify the performance on iPad to see if it reduced significantly.  I ran three trials with and without this flag set, measured the results using the ```neuronProfiler=1``` flag, and found no performance degradation.  I'll remove the flag and move forward with the next RC.
